src/main/java/com/PBA/authservice/persistance/model/dtos/ActiveUserDto.java

```diff
@@ -5,15 +5,14 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import java.sql.Timestamp;
 import java.util.UUID;
 
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
 @Data
-public class PendingUserRequest {
+public class ActiveUserDto {
```

Comment:
```suggestion
public class UserDto {
```

We know it's active.